eng/pipelines/pr.yml

```diff
@@ -134,25 +190,26 @@ stages:
           agentPool: VSEngSS-MicroBuild2022-1ES
         testType: Functional
         testTargetFramework: net8.0
-        timeoutInMinutes: 60
+        testProjectName: Dotnet.Integration.Test
+        timeoutInMinutes: 45
 
 - ${{ if eq(parameters.RunFunctionalTestsOnWindows, true) }}:
   - stage:
-    displayName: Functional Tests on Windows (.NET Core 3.1)
+    displayName: NuGet.Packaging.FuncTest on Windows (.NET 8.0)
     dependsOn: []
     jobs:
     - template: pr.job.yml
       parameters:
-        displayName: Functional Tests on Windows (.NET Core 3.1)
+        displayName: NuGet.Packaging.FuncTest on Windows (.NET 8.0)
         osName: Windows
         ${{ if eq(variables['System.TeamProject'], 'public') }}:
           agentPool: NetCore-Public
           agentDemands: ImageOverride -equals Windows.VS2022Preview.Amd64.Open
         ${{ else }}:
           agentPool: VSEngSS-MicroBuild2022-1ES
         testType: Functional
-        testTargetFramework: netcoreapp3.1
-        timeoutInMinutes: 60
```

Comment:
Just making sure this drop of the timeout is intentional. I'm guessing because net8 runs much faster.

Reply:
That is correct, it always finishes in under 15 minutes which is the default timeout